packages/cfd/src/Containers/dynamic-leverage/dynamic-leverage-market-card.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Table, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { TDynamicLeverageMarketCardProps } from 'Containers/props.types';
+
+export const DynamicLeverageMarketCard = ({
+    market,
+    leverage,
+    market_example,
+    dynamicLeverages,
+}: TDynamicLeverageMarketCardProps) => (
+    <div className='dynamic-leverage-modal__market'>
+        <div className='dynamic-leverage-modal__market-title'>
+            <Text data-testid='market_title' size='xs' weight='bolder' align='center'>
+                {localize(market)}
+            </Text>
+            {market_example ? (
+                <Text
+                    data-testid='example_title'
+                    size='xxxs'
+                    align='center'
+                    className='dynamic-leverage-modal__market-example'
+                >
+                    {localize(market_example)}
+                </Text>
+            ) : null}
+            <Text data-testid='leverage_title' size='xxs' color='red' align='center'>
+                {localize(leverage)}
+            </Text>
+        </div>
+        <Table className='dynamic-leverage-modal__market-table'>
+            <Table.Header>
+                <Table.Row className='dynamic-leverage-modal__market-table-header-row'>
+                    <Table.Head>
```

Comment:
@arshad-rao-deriv please map through 3 <Table.Head>s below based on their content, they are identical. :)

Reply:
Updated. I extracted out it to a function outside the component.